stdlib/LinearAlgebra/test/blas.jl

```diff
@@ -458,7 +458,44 @@ Base.setindex!(A::WrappedArray, v, i::Int) = setindex!(A.A, v, i)
 Base.setindex!(A::WrappedArray{T, N}, v, I::Vararg{Int, N}) where {T, N} = setindex!(A.A, v, I...)
 Base.unsafe_convert(::Type{Ptr{T}}, A::WrappedArray{T}) where T = Base.unsafe_convert(Ptr{T}, A.A)
 
-Base.stride(A::WrappedArray, i::Int) = stride(A.A, i)
+Base.strides(A::WrappedArray) = strides(A.A)
+
+@testset "strided interface adjtrans" begin
+    x = WrappedArray([1, 2, 3, 4])
+    @test stride(x,1) == 1
+    @test stride(x,2) == stride(x,3) == 4
+    @test strides(x') == strides(transpose(x)) == (4,1)
+    @test pointer(x') == pointer(transpose(x)) == pointer(x)
+    @test_throws BoundsError stride(x,0)
+
+    A = WrappedArray([1 2; 3 4; 5 6])
+    @test stride(A,1) == 1
+    @test stride(A,2) == 3
+    @test stride(A,3) == stride(A,4) == 6
```

Comment:
Whoops, this needs to change too.

```suggestion
    @test stride(A,3) == stride(A,4) >= 6
```